NExtends/Expressions/ExpressionMerger.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace NExtends.Expressions
+{
+	public class ExpressionMerger : ExpressionVisitor
+	{
+		Expression CurrentParameterExpression { get; set; }
+
+		public static Expression<Func<TIn, TOut>> Merge<TIn, TA, TOut>(Expression<Func<TIn, TA>> entryPoint, Expression<Func<TA, TOut>> expression1)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of generic parameters in the 'ExpressionMerger.Merge' method to no more than the 2 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2436)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'System.Linq.Expressions.LambdaExpression' instead of 'System.Linq.Expressions.Expression<System.Func<TA, TOut>>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3242)

Reply:
@rducom on peut passer à 3 arguments max cette règle ? Dans RDD sur les controller je crois qu'on est déjà à 3. Ca permettra de régler le pb ici, et à qq autres endroits aussi ;)

Pour les Lambda, je trouve que ça fait perdre un peu de contexte à la méthode, mais j'imagine qu'on ne peut pas faire sauter la règle QUE dans le cas des Lambda si ?

Reply:
@rducom up pour passer à 3 ? ou c'est imposé par Sonarcloud ?